Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -1999,7 +1999,7 @@ extension SwiftObjectServerTests {
 
     // MARK: - Combine promises
 
-    func testEmailPasswordAuthentication() {
+    func testCombineEmailPasswordAuthentication() {
```

Comment:
It's best if these test end with `Combine` instead of begin with it. E.g.,: `testEmailPasswordAuthenticationCombine`. If we are looking for the test via searching in Xcode, we likely don't want all Combine tests to come up– we are probably looking for a specific set of tests.